stix_shifter_modules/azure_sentinel/stix_translation/json/to_stix_map.json

```diff
@@ -30,19 +30,32 @@
       "key": "x-msazure-sentinel-alert.assignedTo",
       "object": "alert"
     },
-    "category": {
-      "key": "x-msazure-sentinel-alert.category",
-      "object": "alert"
-    },
+    "category": [
+      {
+      "key": "x-ibm-finding.finding_type",
+      "object": "finding"
+      },
+      {
+      "key": "x-oca-event.category",
+      "object": "event"
+      }
+    ],
     "closedDateTime": {
-      "key": "x-msazure-sentinel-alert.closedDateTime",
-      "object": "alert"
+      "key": "x-ibm-finding.closedDateTime",
+      "object": "finding"
```

Comment:
I think `closedDateTime` should be mapped to `last_observed`
```
"closedDateTime": {
      "key": "last_observed",
      "cybox": false
    }
```

Reply:
fixed, see https://github.com/opencybersecurityalliance/stix-shifter/blob/59997e2197fbe07a524f8797d3b11677afc336d9/stix_shifter_modules/azure_sentinel/stix_translation/json/to_stix_map.json#L39 

https://github.com/opencybersecurityalliance/stix-shifter/blob/59997e2197fbe07a524f8797d3b11677afc336d9/stix_shifter_modules/azure_sentinel/stix_translation/json/to_stix_map.json#L2